README.md

```diff
@@ -103,7 +103,7 @@ This will bring up etcd listening on port 2379 for client communication and on p
 Next, let's set a single key, and then retrieve it:
 
 ```
-etcdctl put mykey "this is awesome"
+etcdctl set mykey "this is awesome"
```

Comment:
@sntdevco `put` is correct. 
https://github.com/etcd-io/etcd/tree/master/etcdctl#put-options-key-value
`set` is v2 - https://github.com/etcd-io/etcd/blob/master/etcdctl/READMEv2.md